Tools/peg_generator/scripts/test_pypi_packages.py

```diff
@@ -72,7 +72,7 @@ def run_tests(dirname: str, tree: int, extension: Any) -> int:
         skip_actions=False,
         tree_arg=tree,
         short=True,
-        extension=extension,
+        extension=None,
```

Comment:
Hm, looks like the `extension` argument is non-functional now, so why not delete it completely (and also from the called function)?